CONTRIBUTING.md

```diff
@@ -122,3 +122,27 @@ The command `Developer: Toggle Developer Tools` gives access to classic develope
 `yarn run test-it`
 
 It is launching UI tests. Beware that it can take several minutes to start. Stay tuned for improvements to come later.
+
+## How to provide a new release version on VS Code Marketplace
+
+* Check that the version in package.json has not been published yet
+    * If already published:
+        * Update version in `package.json`
```

Comment:
For this and for creating a tag it might be beneficial to look at the script that María has just added for `kaoto-ui`: https://github.com/KaotoIO/kaoto-ui/pull/943

But that is just a suggestion in case you think it would be useful.

Reply:
reported https://github.com/KaotoIO/vscode-kaoto/issues/94